pkgs/applications/virtualization/open-vm-tools/default.nix

```diff
@@ -23,6 +23,18 @@ stdenv.mkDerivation rec {
   buildInputs = [ fuse glib icu libdnet libmspack libtirpc openssl pam procps rpcsvc-proto xercesc ]
       ++ lib.optionals withX [ gdk-pixbuf-xlib gtk3 gtkmm3 libX11 libXext libXinerama libXi libXrender libXrandr libXtst ];
 
+  patches = [
+    # Fix building with glib 2.68. Remove after next release.
+    # We drop AUTHORS due to conflicts when applying.
+    # https://github.com/vmware/open-vm-tools/pull/505
+    (fetchpatch {
+    url = "https://github.com/vmware/open-vm-tools/commit/82931a1bcb39d5132910c7fb2ddc086c51d06662.patch";
```

Comment:
The formatting of this is a tiny bit off, but otherwise this LGTM.

Reply:
Ack, was a bit too quick and didn't spot the mistake. Fixed.